mmv1/products/oracledatabase/AutonomousDatabase.yaml

```diff
@@ -195,6 +195,7 @@ properties:
       - name: 'privateEndpointIp'
         type: String
         description: 'The private endpoint IP address for the Autonomous Database. '
+        output: true
```

Comment:
I don't think `Output: true` is needed here. 
1. `default_from_api: true` means the field is Computed + Optional. `Output: true` means the field is computed. Withe the existence of `default_from_api: true`, `Output: true` is redundant.


2. The field `private_endpoint_ip` can be retrieved  and used in additional resources now. Do you see any issues? 
 
Thanks.